packages/app/src/app/components/WorkspaceSelect/WorkspaceSelect.tsx

```diff
@@ -30,8 +30,8 @@ export const WorkspaceSelect: React.FC<WorkspaceSelectProps> = React.memo(
     const history = useHistory();
     const {
       isLegacyFreeTeam,
-      isLegacyPersonalPro,
```

Comment:
Can't we remove the `isLegacyPersonalPro` check from the hook?

Reply:
I want to leave it as a reference and it is still used to conditionally render some components in the pro page for example. it will be removed with the work starting on Monday